src/actors.js

```diff
@@ -247,10 +291,12 @@ export default class Actors extends Resource {
      *
      * The endpoint supports pagination using limit and offset parameters and it will not return more than 1000 array elements.
      *
+     * For more details see (get list of runs endpoint)[https://docs.apify.com/api/v2#/reference/actors/run-collection/get-list-of-runs]
+
      * @memberof ApifyClient.actors
      * @instance
      * @param {Object} options
-     * @param {String} options.actorId - Unique actor ID
+     * @param {String} options.actorId - Actor ID or a tilde-separated owner's username and actor name.
```

Comment:
Actually `/` separated. That's why the `replaceSlashWithTilde` function is there. Ex.: `apify/web-scraper`